src/microbe_stage/OrganelleLayout.cs

```diff
@@ -96,6 +99,21 @@ public bool IsTouchingExistingHex(T organelle)
         return false;
     }
 
+    /// <summary>
+    ///   Returns true if the specified organelle is replacing the last hex of cytoplasm.
+    /// </summary>
+    public bool IsReplacingLast(T organelle)
+    {
+        var replacedOrganelle = GetOrganelleAt(organelle.Position);
```

Comment:
The count check should be before this line. As replacedOrganelle won't be needed in that case.

Reply:
Switched in f9ff25e